new_network_config_example.yaml

```diff
@@ -0,0 +1,234 @@
+      # network.defaultInterface.ipv4[0] -- array of Ipv4Net objects: ipv4.addr, ipv4.netmask
+      # network.defaultInterface.ipv6[0] -- array of Ipv6Net objects: ipv6.addr, ipv6.netmask
+      # network.defaultInterface.mac_addr -- MacAddr object
+      # network.defaultInterface.transmit_speed -- number
+      # network.defaultInterface.receive_speed -- number
+      # network.defaultInterface.dns_servers[0] -- array of IpAddr objects
+      # network.defaultInterface.default -- boolean, whether this is the default interface
+      
+      # network.defaultGateway -- the default gateway of the device
+      # network.defaultGateway.mac_addr -- MacAddr object
+      # network.defaultGateway.ipv4[0] -- array of Ipv4Addr objects
+      # network.defaultGateway.ipv6[0] -- array of Ipv6Addr objects
+      # network.defaultGateway.ssid -- Windows Only for now, string
+      # network.defaultGateway.signal_strength_percent -- Windows Only for now, number 0-100 representing % strength 
+      # network.defaultGateway.connected -- Windows Only for now, boolean
+      
+      # network.interfaces -- array of all network interfaces of the device
+      # network.interfaces[0].name -- string
+      # network.interfaces[0].friendly_name -- string
+      # network.interfaces[0].description -- string
+      # network.interfaces[0].if_type -- Interface Type, Options: Unknown, Ethernet, TokenRing, Fddi, BasicIsdn, PrimaryIsdn, Ppp, Loopback, Ethernet3Megabit, Slip, Atm, GenericModem, FastEthernetT, Isdn, FastEthernetFx, Wireless80211, AsymmetricDsl, RateAdaptDsl, SymmetricDsl, VeryHighSpeedDsl, IPOverAtm, GigabitEthernet, Tunnel, MultiRateSymmetricDsl, HighPerformanceSerialBus, Wman, Wwanpp, Wwanpp2, Bridge
+      # network.interfaces[0].ipv4[0] -- array of Ipv4Net objects: ipv4.addr, ipv4.netmask
+      # network.interfaces[0].ipv6[0] -- array of Ipv6Net objects: ipv6.addr, ipv6.netmask
+      # network.interfaces[0].mac_addr -- MacAddr object
+      # network.interfaces[0].transmit_speed -- numebr
+      # network.interfaces[0].receive_speed -- number
+      # network.interfaces[0].dns_servers[0] -- array of IpAddr objects
+      # network.interfaces[0].default -- boolean, whether this is the default interface
+
+      # --------- Signal strength explanation ---------
+      # Signal strength is generally measured in dBm.
+      # The new_network provider returns signal strength as a percentage, where
+      # 100% represents a signal strength of -50 dBm or better, and 0% represents
+      # a signal strength of -100 dBm or worse. The calculation method is dBm = percentage / 2 - 100.
+      # Signal strengths of ~-67dBm (~66%) or more are considered very good.
+      # Signal strengths of ~-80dBm (~40%) or less are considered not good.
+      # Signal strengths of ~-90dBm (~20%) or less considered unusable.
+      template: |
```

Comment:
Could you yoink this element and add it to the default config instead? Then we can just delete this file and add the above to the readme at some point